targets/TARGET_STM/README.md

```diff
@@ -503,6 +503,22 @@ https://github.com/ARMmbed/mbed-os/blob/master/connectivity/drivers/emac/TARGET_
 Option is also to define your own `HAL_ETH_MspInit` function,
 you then have to add **USE_USER_DEFINED_HAL_ETH_MSPINIT** macro.
 
+#### Custom IRQ Handler and Callback from user application
+To use the custom IRQ Handler and the callbacks, you need to add
+**USE_USER_DEFINED_HAL_ETH_IRQ_CALLBACK** macro
```

Comment:
@pan- 
This is a unique requirement for the customer according to STM, he can add this in the target or the application outside the code tree of mbed-os in mbed_app.json or in ./mbed-os/targets/targets.json. If we add in mbed_lib.json this leads to confusion because same filename in different folders.

Reply:
@pan- I agree, but maybe reason is to make the same "workaround" as for previous USE_USER_DEFINED_HAL_ETH_MSPINIT macro.
When we implemented this, we were less MBED aware.. :-(